src/main/java/com/google/devtools/build/lib/syntax/MethodLibrary.java

```diff
@@ -170,14 +186,45 @@ private static boolean hasElementWithBooleanValue(
             type = Object.class,
             doc = "This collection.",
             // TODO(cparsons): This parameter should be positional-only.
-            legacyNamed = true)
+            legacyNamed = true),
+        @Param(
+            name = "key",
+            doc = "The key to sort with.",
```

Comment:
Imagine a user wondering what value to pass here: what text would help them? Perhaps something like:
"an optional function applied to each element before its truth-value is assessed"

Reply:
(Comment updated: I was confused by the GitHub diff rendering, which made it look as if were in the  'all' function, which cares about truth values.)